qelem/lib/screens/Home.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+import 'package:qelem/common/AppPalette.dart';
+import 'package:qelem/common/Constants.dart';
+import 'package:qelem/common/QelemTheme.dart';
+import 'package:qelem/widgets/HomeAppBar.dart';
+
+class Home extends StatelessWidget {
+  const Home({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return const HomePage();
+  }
+}
+
+class HomePage extends StatefulWidget {
+  const HomePage({Key? key}) : super(key: key);
+
+  @override
+  _HomePageState createState() {
+    return _HomePageState();
+  }
+}
+
+class _HomePageState extends State<HomePage> {
+  int _selectedIndex = 0;
+  String _title = Constants.home;
+  AppBar _appBar = homeAppBar(Constants.home);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _appBar,
+      bottomNavigationBar: bottomNav(),
+    );
+  }
+
+  Widget bottomNav() {
+    return BottomNavigationBar(
+      backgroundColor: AppPalette.qelemPurple,
+      items: const [
+        BottomNavigationBarItem(
+          icon: Icon(Icons.home),
+          label: Constants.home,
+        ),
+        BottomNavigationBarItem(
+          icon: Icon(Icons.help),
+          label: Constants.myQuestions,
+        ),
+        BottomNavigationBarItem(
+          icon: Icon(Icons.person),
+          label: Constants.profile,
+        ),
+      ],
+      currentIndex: _selectedIndex,
+      selectedLabelStyle: QelemTheme.bottomNavSelected,
+      unselectedLabelStyle: QelemTheme.bottomNavUnselected,
+      selectedItemColor: Colors.white,
+      unselectedItemColor: Colors.grey,
```

Comment:
remove these, we can define the bottom nav theme from the Theme in the MaterialApp

Reply:
you r right.